content/en/database_monitoring/setup_postgres/azure.md

```diff
@@ -314,6 +314,7 @@ helm repo update
 helm install <RELEASE_NAME> \
   --set 'datadog.apiKey=<DATADOG_API_KEY>' \
   --set 'clusterAgent.enabled=true' \
+  --set 'clusterChecksRunner: true' \
```

Comment:
I *think* this should be 

```
  --set 'clusterChecksRunner.enabled=true' \
```

But I need to confirm

Reply:
Any update on this? Would like to get this correct so it can be merged in.

Reply:
My above comment is correct, thanks!